packages/core/include/combo/oot/actors/Custom_Torch2.h

```diff
@@ -0,0 +1,32 @@
+#ifndef Z_EN_TORCH2_H
+#define Z_EN_TORCH2_H
+
+#include <combo/common/actor.h>
+
+struct EnTorch2;
+
+typedef struct EnTorch2 {
+    /* 0x000 */ Actor base;
+    /* 0x13c */ ColliderCylinder collider;
+    /* 0x188 */ u8 state;
+    /* 0x189 */ u8 framesUntilNextState;
+    /* 0x18a */ s16 alpha;
+} EnTorch2; // size = 0x18c
```

Comment:
No C++ style comment, no size comment, especially not on a custom struct

Reply:
We don't use C++-style comments.

Reply:
Yes you do.

Reply:
..?

Reply:
![image](https://github.com/OoTMM/OoTMM/assets/22694317/be6f6b79-adda-411f-b41f-c77f3782957e)

Reply:
Most of which you copy-pasted from decomp. Remove it.

Reply:
Done